files/image_config/logrotate/logrotate.d/rsyslog

```diff
@@ -0,0 +1,93 @@
+# These logs should no longer get created. However, in case they do get created,
+# we should keep them to a small size and rotate them also.
+/var/log/mail.info
+/var/log/mail.warn
+/var/log/mail.err
+/var/log/mail.log
+/var/log/daemon.log
+/var/log/kern.log
+/var/log/user.log
+/var/log/lpr.log
+/var/log/debug
+/var/log/messages
+{
+    size 10k
+    rotate 1
+    missingok
+    notifempty
+    compress
+    delaycompress
+    sharedscripts
+    postrotate
+        # calling kill directly instead of 'service rsyslog rotate >/dev/null' due
+        # to bug in init-system-helpers. bug has apparently been fixed in v1.47.
+        # however, debian jessie is still using v1.22.
+        # see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=672218
+        kill -hup $(cat /var/run/rsyslogd.pid)
+    endscript
+}
+
+/var/log/auth.log
+/var/log/cron.log
+/var/log/syslog
+/var/log/teamd.log
+/var/log/quagga/bgpd.log
+/var/log/quagga/zebra.log
+/var/log/swss/sairedis.rec
+/var/log/swss/swss.rec
+{
+    size 512k
```

Comment:
I agree. I was trying to find a balance between reasonable log file size and supporting as small a partition as possible. Increasing the file size to 1M will increase the minimum /var/log partition size by ~8MB, to ~25MB. If you don't see a problem with this, I will make the change and test it ASAP.